test/utils/TransactionSender.py

```diff
@@ -37,7 +28,19 @@ async def send_transaction(self, calls, signers, nonce=None, max_fee=0):
             else:    
                 signatures += list(signer.sign(message_hash))
 
-        return await self.account.__execute__(call_inputs, calldata, nonce).invoke(signature=signatures)
+        return await self.account.__execute__(call_array, calldata, nonce).invoke(signature=signatures)
+
+def from_call_to_call_array(calls):
+    call_array = []
+    calldata = []
+    for i in range(len(calls)):
+        if len(calls[i]) != 3:
+            raise Exception("Invalid call parameters")
+        call = calls[i]
+        entry = (call[0], get_selector_from_name(call[1]), len(calldata), len(call[2]))
```

Comment:
it's a very standard pattern to use `enumerate` when iterating over both indexes and items

```suggestion
    for i, call in enumerate(calls):
        assert len(call) == 3, "Invalid call parameters"
        entry = (call[0], get_selector_from_name(call[1]), len(calldata), len(call[2]))
```